Sources/PT.PM.Common/UstConstantFolder.cs

```diff
@@ -74,16 +74,15 @@ private FoldResult TryGetOrFold(Ust ust)
 
         private FoldResult TryFoldArrayCreationExpression(ArrayCreationExpression arrayCreationExpression)
         {
-            if (arrayCreationExpression.Type?.TypeText == "char" &&
-                arrayCreationExpression.Initializers.Count > 0 &&
-                (bool)arrayCreationExpression.Initializers?.All(i => i is StringLiteral))
+            if (arrayCreationExpression.Initializers?.Count > 0 &&
+                !arrayCreationExpression.Initializers.Any(i => !(i is StringLiteral)))
```

Comment:
To me, `All is StringLiteral` is clearer than `Not any not is StringLiteral`.

Reply:
Agree.